ChangeLog

```diff
@@ -1,3 +1,4 @@
+UPGRADE: Java from 8 to 11
```

Comment:
```suggestion
UPGRADE: Java from 8 to 11 in Dockerfile
```

The other UPGRADE below refer to pom.xml and maybe the difference could be important